how-to-docs/README.md

```diff
@@ -2,7 +2,7 @@
 This folder contains a collection of how-to guides for both Developers and DevOps to use.
 
 - Contents:
-  - [Making an Artifactory robot](artifact-robot.md)
+  - [Making an Artifactory access token](artifactory-token.md)
```

Comment:
This may have broken the link on the hub that we use to direct users to this guide – please could you check and fix if needed?

Reply:
As far as I can see, this guide isn't actually linked on the hub. The hub does link to the support request for requesting Artifactory robot accounts (which would be inconsistent with the language in this updated version and would need to be changed; unless we want to keep calling them robot accounts?).

Reply:
Ah of course, this is more like an internal how-to guide for the ACP team (which probably begs the question: should the guides say they are for ACP team only? Otherwise users of the platform looking at it could get confused and think they have to carry out the instructions).

Reply:
Yes, we should keep the language consistent :)

Reply:
I think it should for the exact reason you mentioned. I'll add a note saying that.